frontend/src/pages/admin-blogs.tsx

```diff
@@ -0,0 +1,50 @@
+import PenIcon from '@/assets/svg/pen-icon';
+import TrasnIcon from '@/assets/svg/trash-icon';
+import AdminSidebar from '@/components/admin-sidebar';
+import { imageUrls } from '@/constants/images';
+
+const AdminBlogs = () => {
+  return (
+    <>
+      <div className="relative flex flex-grow flex-col bg-light dark:bg-dark md:flex-row">
+        <AdminSidebar />
+        <div className="w-full p-3 px-5 md:p-12">
```

Comment:
Adding AdminSidebar in all the blogs will be redundant and not a good practice. So we would be needing the wrapper